content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,212 @@
+### How to use symbolic maths toolbox in Matlab to analyze Fourier series
+### Introduction
+Symbolic math toolbox provides an easy, intuitive and complete environment to interactively learn and apply math operations such as calculus, algebra, and differential equations. It can also perform common analytical computation such as differentiation and integration to get close form results, simplify and manipulate expression for great insights and solve algebraic and differential equations. 
+In this tutorial, the Fourier series is implemented and simulated using Symbolic Math's Toolbox of Matlab(Both forms of the Fourier series, Trigonometric and Exponential, are implemented). The proposed programs are versatile and can receive any function of time(t). It means the function is dependent on time. Moreover, the program gives plots of harmonics, original and approximated functions, magnitude spectrum, and phase spectrum. The advantage is that this toolbox is already available in Matlab. You can check [here] to understand more how the Fourier series works mathematically; you can check [here](https://www.mathsisfun.com/calculus/fourier-series.html).
+
+### ### Prerequisites
+To follow along with this tutorial, you will need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [MATLAB basics](/engineering-education/getting-started-with-Matlab/).
+
+### Symbolic math toolbox
+This toolbox has a wide range of applications:
+1. You can use this toolbox to visualize analytical expressions in 2-D and 3-D and even animate plots to create videos from it.
+2. Symbolic math in the live editor(mode in Matlab) lets you interactively update and display Symbolic math computations. Also, Matlab code formatted text, equations, and images can be published as executable live scripts, PDFs, or HTML documents. While working with analytical problems, you can get suggestions for the next step. These suggestions help you insert and execute function calls or live editor tasks directly into live scripts.
+3. In addition to providing exact rational math, the Symbolic toolbox provides precision for higher or lower positions, allowing algorithms to run higher or lower than Matlab's in-built double. It also includes units for working with physical quantities and performing dimensional analysis.
+
+![Units for working with physical quantities](/engineering-education/how-to-use-symbolic-math's-toolbox-in-Matlab-to-analyze-the-Fourier-series/symbolic-one.jpeg)
```

Comment:
It seems you missed my point on the word MATLAB being in upper case. It should be MATLAB, not Matlab. And this does not apply to the branch name.